R/stan_glm.fit.R

```diff
@@ -455,11 +455,11 @@ stan_glm.fit <- function(x, y, weights = rep(1, NROW(x)),
 }
 
 
-# Add extra level _NEW_ to each group
-# 
-# @param Ztlist ranef indicator matrices
-# @param cnms group$cnms
-# @param flist group$flist
+#' Add extra level _NEW_ to each group
+#' 
+#' @param Ztlist ranef indicator matrices
+#' @param cnms group$cnms
+#' @param flist group$flist
```

Comment:
I think this shouldn't have roxygen comments `#'`. Just `#` or it will try to create a doc page for this.